src/pages/orders/OrderDetails.js

```diff
@@ -13,27 +13,14 @@ import {
   Button,
 } from "reactstrap";
 
-import DateFormat from "../../utils/dateFormat";
 import BackButton from "../../components/BackButton/BackButton";
+import OrderDetailsOrder from "../../components/OrderDetails/OrderDetailsOrder";
+import OrderDetailsBuyer from "../../components/OrderDetails/OrderDetailsBuyer";
+import OrderDetailsStatus from "../../components/OrderDetails/OrderDetailsStatus";
 
```

Comment:
Just put the components inside the orders directory as these components are not shared.